### ISSUE TITLE
sholazar basin ,oracle, daily : Song of Wind and Water

### ISSUE THREAD

**Issue by plantemarc** [dalaranwow/dalaran-wow#5595]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1. sholazar basin ,oracle daily Song of Wind and Water
2. using Tempest Drums at the Stormwright's Shelf devour 3 Storm Revenants with no issue but when i try on Aqueous Spirits. always said invalid targer tried at 100% 50% 30% 20% 10% 5%health
3. 
![wowscrnshot_090418_202650](https://user-images.githubusercontent.com/42881489/45064513-44203a80-b082-11e8-8236-2b94b7289044.jpg)

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by galgnu** [comment]
I done the quest without issues. 
It works where first you devour a storm spirit after which you change form. In the new form you attack a water spirit and bring it to 20% or what the limit is then you devour it. Repeat 3 times. 
So first air -> water -> air -> water etc. 
Dont exit the mount/mob you transform into before you have killed a water elemental.

**Comment by Golganeth** [comment]
It seems to work just fine, so I'm closing
